samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -0,0 +1,493 @@
+   * Blinks at a 1 second interval if the recovery mode is active on the device and the identification mode is inactive on the device.
+
+Button 1:
+   Toggles between different modes of the Fast Pair advertising set:
+
+   * Without an Account Key stored on the device:
+
+     * Fast Pair advertising disabled.
+     * Fast Pair discoverable advertising (the default after the system bootup).
+
+   * With an Account Key stored on the device:
+
+     * Fast Pair advertising disabled.
+     * Fast Pair not discoverable advertising (the default after the system bootup).
+
+   .. note::
+      The Bluetooth advertising is active only until the Fast Pair Provider connects to a Bluetooth Central.
+      After the connection, you can still switch the advertising modes, but the switch will come into effect only after disconnection.
+
+      The sample automatically switches to the following Fast Pair advertising modes under certain conditions:
+
+      * Fast Pair not discoverable advertising - On the Account Key write operation.
+      * Fast Pair advertising disabled:
+
+        * Right before the factory reset operation.
+          Press **Button 1** to set the device to the Fast Pair discoverable advertising mode after the unprovisioning operation.
+        * After the beacon clock synchronization.
+
+Button 2:
+   Stops the ongoing ringing action.
+
+Button 3:
+   Decrements the battery level by 10% (the default value in the :kconfig:option:`CONFIG_APP_BATTERY_LEVEL_DECREMENT` Kconfig option), starting from the full battery level of 100%.
+   In the default sample configuration, with the :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_BATTERY_DULT` Kconfig option enabled, pressing the button when the battery level is lower than the decrement value transitions the device back to the starting point of 100%.
+   You can disable the default :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_BATTERY_DULT` Kconfig setting to transition through the mode in which the battery level indication is not supported (refer to the last point in the list).
+   The battery level is encoded in the FMDN advertising set according to the following rules:
+
+   * Normal battery level (starting point, as derived from the 100% battery level) - The battery level is higher than 40% and less than or equal to 100%.
+     The lower threshold, 40%, is controlled by the :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_BATTERY_LEVEL_LOW_THR` Kconfig option.
+   * Low battery level - The battery level is higher than 10% and less than or equal to 40%.
+     The lower threshold, 10%, is controlled by the :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_BATTERY_LEVEL_CRITICAL_THR` Kconfig option.
+   * Critically low battery level (battery replacement needed soon) - The battery level is higher than or equal to 0% and less than or equal to 10%.
+   * Battery level indication unsupported - Occurs when the special :c:macro:`BT_FAST_PAIR_FMDN_BATTERY_LEVEL_NONE` value is used.
```

Comment:
The  macro don't link. Could you check if its correct?

Reply:
It is probably because I did not upstream yet the service documentation which adds API elements from our header file

Reply:
I unresolved this comment so I can double check if this is fixed once I add the service documentation.

Reply:
this link also started to work in the follow-up PR:

https://github.com/nrfconnect/sdk-nrf/pull/15729
https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-15729/nrf/samples/bluetooth/fast_pair/locator_tag/README.html